### PR TITLE
Add Electric & Gas Bill Estimates to Building Details

### LINKED ISSUE #48: Add Price Estimates for Electricity and Gas Use
Since all properties are in Chicago, we could use an average price per therm and do some calculations to show how much we think a property is spending on gas and electric use. This would be useful for activists to be able to pitch to building owners that they are spending way more than other buildings.

We may even be able to make an estimated cost per square foot!

A helpful resource: https://www.bls.gov/regions/midwest/news-release/AverageEnergyPrices_Chicago.htm

### 2021 Data

Using 2021 data from the source above because it's our latest year data, we could put that in a constant

July 2021 Chicago gas price per **Therm: $1.101**
July 2021 Chicago electric price **per kWh: $0.147**

### PR DESCRIPTION
**Pull request by vkoves**
Adds in price estimates for the gas and electric stats, using Chicago retail average prices. Here's an example from Herman Hall:

Screenshot from 2023-12-03 12-50-18

Resolves #48

### CODE REVIEW

**Comment by vkoves** [comment]
Thoughts on this:

- Move "for 2021" into main emissions & energy info title to make it more prominent
- Indicate price estimate is per year/for that year (e.g. "$X for 2021")
- Add explainer for how we get this number

Screenshot from 2023-11-28 20-38-36

File: src/common-functions.vue [hunk 1]
```diff
@@ -184,4 +184,31 @@ export function getOverallRankEmoji(
 
   return null;
 }
+
+export const UtilityCosts = {
+  year: 2021,
+  source: 'https://www.bls.gov/regions/midwest/news-release/AverageEnergyPrices_Chicago.htm',
```
Comment (vkoves) [inline src/common-functions.vue:190]: This is the actual data source, let's make sure to note this in the data explainer

**Review by alexkcode — COMMENTED** [review]
Looks pretty straight forward to me. I'm assuming this has been tested already? I can do a quick test myself when I get the chance.

**Review by alexkcode — APPROVED** [review]
Checked to be working in cases where there is zero or otherwise exception gas usage and cases where there is old data (like United Center). I don't know that I would include the PR template in this change as well but doesn't seem like a big issue here.

File: src/common-functions.vue [hunk 2]
```diff
@@ -184,4 +184,51 @@ export function getOverallRankEmoji(
...
+
+/**
+ * A constant for Chicago avg. utility costs in the latest data year, which we use to estimate an
+ * upper bound of how much a building would pay for gas and electric
+ */
+export const UtilityCosts = {
+  year: 2021,
+  source: 'https://www.bls.gov/regions/midwest/news-release/AverageEnergyPrices_Chicago.htm',
+  electricCostPerKWh: 0.147,
+  gasCostPerTherm: 1.101,
```
Comment (vkoves) [inline src/common-functions.vue:196]: This should be `1.192`

Screenshot from 2023-12-05 20-43-32

File: src/common-functions.vue [hunk 3]
```diff
@@ -184,4 +184,51 @@ export function getOverallRankEmoji(
...
 }
+
+/**
+ * A constant for Chicago avg. utility costs in the latest data year, which we use to estimate an
+ * upper bound of how much a building would pay for gas and electric
+ */
+export const UtilityCosts = {
+  year: 2021,
+  source: 'https://www.bls.gov/regions/midwest/news-release/AverageEnergyPrices_Chicago.htm',
+  electricCostPerKWh: 0.147,
```
Comment (vkoves) [inline src/common-functions.vue:195]: Should be `0.143`

Screenshot from 2023-12-05 20-44-46

**Review by alexkcode — APPROVED** [review]
Re-approving. Went through manual check of conversion to therms units for bill calculation and discussion of the rounding on number being displayed.

**Pull request merged**